varats/varats/projects/perf_tests/feature_perf_cs_collection.py

```diff
@@ -34,7 +34,7 @@ class FeaturePerfCSCollection(VProject):
         bb.source.Git(
             remote="https://github.com/se-sic/FeaturePerfCSCollection.git",
             local="FeaturePerfCSCollection",
-            refspec="origin/HEAD",
+            refspec="origin/ba-messerig",
```

Comment:
With this change, the FeaturePerfRunner runs on your branch.
```suggestion
            refspec="origin/HEAD",
```